index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      header__buttoncontent="width=device-width, initial-scale=1.0"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20,500,0,-25"
+    />
+    <title>Medical core</title>
+  </head>
+  <body>
+    <!-- start navbar -->
+    <nav class="header">
+      <div class="menu">
```

Comment:
I suggest you make this menu class an independent block or otherwise, you suggest you make it element of the header header.